modules/monitoring-alertmanager-default-configuration.adoc

```diff
@@ -25,5 +26,6 @@ The default configuration of the {product-title} Monitoring Alertmanager cluster
   - name: deadmansswitch
 ----
 
-// FIXME check that this is up-to-date
+{product-title} monitoring ships with a "Dead man’s switch" that always triggers by default to ensure the availability of the monitoring infrastructure.
```

Comment:
default alert is Watchdog now, see 
https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/rules.yaml#L889-L899